static/app/components/segmentedControl.tsx

```diff
@@ -323,6 +324,7 @@ const VisibleLabel = styled('span')<{
   font-weight: ${p => (p.isSelected ? 600 : 400)};
   letter-spacing: ${p => (p.isSelected ? '-0.015em' : 'inherit')};
   text-align: center;
+  line-height: ${p => p.theme.text.lineHeightBody};
```

Comment:
Slipping this in…

**Before ——** descenders (e.g. hook at the bottom of the letter g) are cropped
<img width="82" alt="Screenshot 2023-02-01 at 10 55 53 AM" src="https://user-images.githubusercontent.com/44172267/216136989-b86b50dc-9c15-432a-bf28-31099c28f006.png">

**After ——**
<img width="82" alt="Screenshot 2023-02-01 at 10 55 19 AM" src="https://user-images.githubusercontent.com/44172267/216136889-5aed151a-391d-4e5f-a07b-332367222208.png">

Reply:
nice